index.bs

```diff
@@ -4308,7 +4365,7 @@ In these cases, the [=authenticator=] provides no guarantees about its operation
 
 <figure id="fig-attStructs">
     <img src="images/fido-attestation-structures.svg"/>
```

Comment:
This figure should be updated to include the BE/BS flags pictured, just like UV/UP. I realise that this comment is not particularly relevant to DPK, however the PR containing changes for BE/BS has already landed, and short of creating a brand new issue this seemed like the next best place to report it...

Reply:
Good catch! That indeed belongs in a new issue, I'll open one.

Reply:
Moved to issue #1775 and PR #1776.